include/constants.hpp

```diff
@@ -1,6 +1,9 @@
 #pragma once
 
+#include <stdint.h>
```

Comment:
Why do you need this here?

Reply:
added to resolve the error - unknown type name 'uint8_t'
Got above error when I included constants.hpp in the utest*.cpp file.

Reply:
But why would this be needed here? Could you please check if the requirement us in constant.hpp or in the test file.